head/src/test/kotlin/io/qalipsis/core/head/report/DataSeriesServiceImplTest.kt

```diff
@@ -96,7 +96,7 @@ internal class DataSeriesServiceImplTest {
     private lateinit var dataSeriesConverter: DataSeriesConverter
 
     @Test
-    internal fun `should create the data series with the default operation and no field name`() =
+    internal fun `should create the data series with the default operation, no field name and color opacity of 5`() =
```

Comment:
Since there is no color in that test, the color opacity can be removed as well. We try to create with a minimal definition here.